### ISSUE TITLE
Craft welcome message, FAQ, and workflow for new volunteers

### ISSUE THREAD

**Issue by miguelrios** [SoPR/maria-tech-brigade-effort#19]
We have a lot of people coming to the Slack, which is awesome! But we need a better way to channel them efficiently towards whatever is needed and they can help with. 

I propose writing a welcome message with a simple FAQ:

- Where to start?
- How can you help? 
- What channels should I subscribe to?
- etc...

We can create a more specific workflow with steps to onboard new volunteers too. 

Thoughts?

**Comment by jsuarezatzendesk** [comment]
Yes! We have the #maria-volunteer channel which is working on some of this

Do you have any specific ideas or things you want to include?

**Comment by miguelrios** [comment]
@jsuarezatzendesk the basic questions above, as well as a good description of every channel we're creating. Thanks for working on this! I'll assign it to you then :)

**Comment by miguelrios** [comment]
@jsuarezatzendesk can't assign it to you for some reason. Feel free to assign the ticket to yourself.

**Comment by jsuarezatzendesk** [comment]
I think because I'm not in the SoPR team

**Comment by jsuarezatzendesk** [comment]
I can't assign it to myself, either

**Comment by miguelrios** [comment]
Sent you an invite: https://github.com/SoPR/maria-tech-brigade-effort/invitations

**Comment by Tokynet** [comment]
We should highlight in a subtle manner that all these efforts are being done as volunteers, folks should provide information like "i can work 3hrs daily" etc... this way folks that are working together have a better understanding on when to expect any "deliverables" etc.

Sadly If I'm mentioning it, its because it has already been an issue, I just want to keep the effort moving forward while keeping folks happy/engaged.
